public/src/js/main.js

```diff
@@ -1,4 +1,3 @@
-/* global System */
```

Comment:
Why was this comment removed?

Reply:
This is a `eslint` global which was being flagged as unused in this file.